client/pages/Event/EventsPage.jsx

```diff
@@ -137,24 +144,100 @@ class EventsPage extends React.Component {
     });
   }
 
+
   /**
-  * @description It renders list of event card
-  *
+   * @description It renders list of event card
+   *
    * @memberof EventsPage
    */
-  renderEventGallery = () => {
-    const { eventList } = this.state;
+  renderEventGallery = (catList) => {
+    const {
+      eventList,
+    } = this.state;
     if (eventList.length) {
       const listOfEventCard = mapListToComponent(eventList, EventCard);
       return (<div className="event__gallery">
         {listOfEventCard}
+        {this.renderCreateEventButton({ categories: catList })}
       </div>);
     }
-    return <EventNotFound statusMessage="404" mainMessage="Events not found" />;
+
+    let actions = [
+      key => this.renderCreateEventButton({
+        categories: catList, message: ADD_EVENT_INSTRUCTION, key,
+      }),
+      key => this.renderOldEventsButton(key)];
+
+    actions = actions.map((action, index) => action(index));
+    return <EventNotFound
+      statusMessage="No upcoming events"
```

Comment:
I think we could use "No event for the selected date and filter". No upcoming events sounds like no event from that point upwards (Like no event even for tomorrow whereas we are referring to that particular day) cc @andela-sjames

Reply:
@chibujax, waiting on @andela-sjames's response on this.

Reply:
I think @chibujax's comment has some bit of weight to it. And also I can see you trying to implement what we had discussed earlier. But I'd say we have "No event for the selected date and filter" until we have the dashboard logic for showing upcoming events